core/src/main/kotlin/model/documentableProperties.kt

```diff
@@ -46,3 +46,17 @@ data class CheckedExceptions(val exceptions: SourceSetDependent<List<DRI>>) : Ex
     }
     override val key: ExtraProperty.Key<Documentable, *> = CheckedExceptions
 }
+
+enum class Language {
+    JAVA, KOTLIN, UNKNOWN
+}
```

Comment:
What do you think about renaming it to something like `SourceLanguage` (preferred) or `DocumentableLanguage` as before? 

My concern is that this enum will be used by someone as a feature/language toggle somewhere on a higher level, and I'd like to try to narrow down the scope of it's usage just to the source/documentable level

If it clashes with `data class SourceLanguage`, it can be renamed to `SourceLanguageExtra`

Reply:
This could still be done, though there might also be a pre-existing alternative that is more comprehensive, perhaps in the compiler.